src/app/shared/components/search-popover/search-popover.component.ts

```diff
@@ -163,6 +171,7 @@ export class SearchPopoverComponent implements OnInit {
         break;
 
       case 'REMOVE':
+        this.countSelectedList = this.countSelectedList -1
```

Comment:
what happens if i remove a session manager? at that time also count should reduce?